package.json

```diff
@@ -26,6 +26,8 @@
     "yargs-parser": ">=18.1.2"
   },
   "resolutions": {
-    "yargs-parser": ">=18.1.2"
+    "yargs-parser": ">=18.1.2",
```

Comment:
Consider removing the old resolution. Chances are, it is no longer required. And having less resolutions is a win in my book!